index.js

```diff
@@ -1,39 +1,266 @@
+        answer3.includes("academy") ||
+        answer3.includes("coding")
+      ) {
+        stateFive();
+        thirdLoop = false;
+      } else if (answer3.includes("bathroom")) {
+        stateFour();
+        thirdLoop = false;
+      } else if (answer3.includes("inv") || answer3 === "i") {
+        showInv();
+        answer3 = await ask("\n");
+      } else {
+        answer3 = await ask("Im sorry, I cannot " + answer3 + "\n");
+      }
+    }
+  }
+  /*
+THIRD ROOM ENDS/FOURTH ROOM BEGINS ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+  async function stateFour() {
+    //THIS WILL BE THE BATHROOM
+    let fourthLoop = true;
+    let answer4 = await ask(
+      "Welcome to the bathroom. There's nothing interesting here. Perhaps you should go back to the hall. \n"
+    );
+    while (fourthLoop) {
+      if (answer4.includes("hall")) {
+        stateThree();
+        fourthLoop = false;
+      } else if (answer4.includes("inv") || answer4 == "i") {
+        showInv();
+        answer4 = await ask("\n");
+      } else {
+        answer4 = await ask("Im sorry, I cannot " + answer4 + "\n");
+      }
+    }
+  }
+  /*
+FOURTH ROOM ENDS/FIFTH ROOM BEGINS ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+  async function stateFive() {
+    //THIS WILL BE THE CLASSROOM
+    let fifthLoop = true;
+    let answer5 = await ask(
+      "Welcome to the Burlington Coding Academy. In the back corner of the room you see Bobs elegant hat. You also see a copy of Sam's Vermont Tech cheat sheet. In the front of the room there is a laptop. VS Code is open. There appears to be a serious bug. Maybe you should fix it, but you will need some things before you fix the bug.\n"
+    );
+    while (fifthLoop) {
+      if (answer5.includes("hall")) {
+        stateThree();
+        fifthLoop = false;
+      } else if (answer5.includes("inv") || answer5 === "i") {
+        showInv();
+        answer5 = await ask("\n");
+      } else if (answer5.includes("take") && answer5.includes("hat")) {
+        inv.hat = "Bob's Elegant Hat";
+        answer5 = await ask(
+          "You have taken Bobs Elegant Coding SuperPowers Hat \n"
+        );
+      } else if (answer5.includes("take") && answer5.includes("cheat sheet")) {
+        inv.vtcs = "Vermont Tech Cheat Sheet";
+        answer5 = await ask("You have taken Sams Vermont Tech Cheat Sheet\n");
+      } else if (
+        answer5.includes("fix bug") &&
+        inv.vtcs == "Vermont Tech Cheat Sheet" &&
+        inv.hat == "Bob's Elegant Hat"
+      ) {
+        console.log(
+          "Using the combined superpowers of Bobs Elegant Hat and Sams Vermont Tech Cheat Sheet you have fixed the bug and won the game!\n");
+        fifthLoop = false;
+        break;
```

Comment:
Breaking here before the next line will ignore the process.exit line of code. Since process.exit will break out of the program itself, let alone the while loop, the break statement isn't needed